src/material/button/button.md

```diff
@@ -33,10 +33,22 @@ screen-readers, which will read the text character-by-character. We leave the de
 approach this to the consuming app.
 
 ### Accessibility
-Angular Material uses native `<button>` and `<a>` elements to ensure an accessible experience by
-default. The `<button>` element should be used for any interaction that _performs an action on the
-current page_. The `<a>` element should be used for any interaction that _navigates to another
-view_.
+Angular Material uses native <button> and <a> elements. Both of the two look the same when applied Material Design Styling, but have different semantic meaning for assitive technology. To ensure an accessible experience, choose the element which is most approprite for the purpose of the interaction.
 
-Buttons or links containing only icons (such as `mat-fab`, `mat-mini-fab`, and `mat-icon-button`)
-should be given a meaningful label via `aria-label` or `aria-labelledby`.
+#### Performing an Action
+Use the `<button>` element for interactions that _perform an action on the current page_.
+ - submitting a form
+ - sending a chat message
+ - copying content to the clipboard
+
+#### Navigation
+Use the `<a>` element for interactions that _navigate to another view_.
```

Comment:
> _navigate to another view_

You use "view" here, and "page" above. Are these synonymous? If so, consider just choose one and sticking with it throughout. Additionally, what exactly defines a view? If I'm on a page with a `mat-tab-group` and I switch tabs, am I navigating to another view? Does the URL need to change? Some guidance here might go a long way (but it can be tricky to author guidance here, so I understand if you want to keep things vague).

Reply:
+1, I see that the original text said "view" which is too ambiguous. IMO we should say that anchors should be used when the page URL would change.

Reply:
> IMO we should say that anchors should be used when the page URL would change.

One exception to this in our app are scenarios where we have components that "sync state" to the URL. For example, a `mat-tab-group` that syncs its state to the URL via matrix params (e.g. `;tab=overview`). In this case, the URL is changing, but tab is the correct role for this interaction. For this reason, I think we now say that links should be used if any URL path segments change (and maybe a subset of query parameter changes as well). It may be worth leaving a note mentioning this.